### PR TITLE
Map page cards

### PR DESCRIPTION
**Pull request by Guy-deMessimy**
component for add in view iss page

### CODE REVIEW

**Review by Marie-Garde — DISMISSED** [review]
Good job

**Comment by LoicBrassart** [comment]
@Marie-Garde (As a sidenote: you left a comment but didn't *approve* the PR ;-) )

File: src/App.css [hunk 1]
```diff
@@ -3,6 +3,9 @@
   margin: 0;
   padding: 0;  
 }
-body {
+.page-container {
+  width: 1800px;
+  height: 2000px;
   background-color:ivory;
+  border: solid 2px black
 }
```
Comment (LoicBrassart) [inline src/App.css:11]: Is this normal ? You want to keep updating this sole CSS file ? :thinking:

**Review by LoicBrassart — APPROVED** [review]
I may have spotted a minor Evil (`App.css`) but otherwise it seems ok to me

**Pull request closed without merging**